### ISSUE TITLE
508-defect-2 [HEADINGS]: Headings should be logically ordered

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#28345]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a screen reader user, I expect headings to accurately represent the hierarchy of the page so I can understand where sections begin and end.

## Details

"Download debt letters" is not logically a part of the section "Current debts," yet is semantically grouped with "Current debts" because it is a `h3` below the `h2` "Current debts." 

If the intention was for "Download debt letters" to indeed be a `h3` grouped under "Current debts", the visual sizing of "Download debt letters" should match `h3` styling, not `h2` styling.

## Acceptance Criteria

- [ ] "Download debt letters" is written as a `h2` OR
- [ ] "Download debt letters" is styled as a `h3`

## Steps to Recreate

1. Log in as user 1 and go to `https://staging.va.gov/manage-va-debt/your-debt/`
2. Open headingsMap and confirm "Download debt letters" is grouped under "Current debts"

## Screenshots or Trace Logs

<img width="1305" alt="Screen Shot 2021-08-05 at 1 08 49 PM" src="https://user-images.githubusercontent.com/14154792/128392296-4069fc8e-93bd-4358-8122-d7297167cc22.png">
